docs/fsharp-cheatsheet.md

```diff
@@ -740,6 +743,112 @@ Another way of implementing interfaces is to use *object expressions*.
 
 *Partial active patterns* share the syntax of parameterized patterns but their active recognizers accept only one argument.
 
+<div id="asynchronous-programming"></div>
+
+## Asynchronous Programming
+
+F# asynchronous programming is centered around two core concepts: async computations and tasks.
+
+    async {
+        do! Async.Sleep (waitInSeconds * 1000)
+        let! asyncResult = asyncComputation
+        use! disposableResult = iDisposableAsyncComputation
+    }
+
+### Async vs Tasks
+
+An async computation is a unit of work, and a task is a promise of a result. A subtle but important distinction.
+Async computations are composable and are not started until explicitly requested; Tasks (when created using the `task` expression) are _hot_:
+
+    let runAsync waitInSeconds =
```

Comment:
while you show how it works in the end, this naming is a really bad example, because it's just creating an instance of a builder.
The thing being created is an Async<T>, and it is created at the end of 768
765 can happen many times - one per instantiation of a running task. I'd use the string 'starting workflow'
I would also consider putting a print before L764 to convey when things start getting created
The other thing that needs to be covered is that before 764, exceptions work on way, and inside the block, it's very different

Reply:
Yes. That was poor wording on my part.